flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/DispatcherTest.java

```diff
@@ -1225,7 +1257,16 @@ public void testOverridingJobVertexParallelisms() throws Exception {
 
     private JobManagerRunner runningJobManagerRunnerWithJobStatus(
             final JobStatus currentJobStatus) {
+        return runningJobManagerRunnerWithJobStatus(currentJobStatus, jobId, 0L);
+    }
+
+    private JobManagerRunner runningJobManagerRunnerWithJobStatus(
+            final JobStatus currentJobStatus, final JobID jobId, long startTime) {
         Preconditions.checkArgument(!currentJobStatus.isTerminalState());
+        long[] stateTimeStampsForRunningJob = new long[JobStatus.values().length];
```

Comment:
I assume that this method ends up being called and we want the order of the jobs to be sensible. Would the most sensible for the UI be the latest first?

Reply:
@davidradl yeah, good point!